sdk/servicebus/azure-messaging-servicebus-stress/pom.xml

```diff
@@ -42,6 +42,12 @@
       <version>2.7.18</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter;external_dependency} -->
     </dependency>
 
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-amqp</artifactId>
+      <version>2.10.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```

Comment:
nit; should this be `<!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->` ?

Reply:
not sure, I'm going to remove it from stress tests since we'd get dependency on newest core-amqp via SB now and we have FromSource pipelines to validate compatibility